src/shell/shell_misc.cpp

```diff
@@ -355,9 +355,9 @@ std::string DOS_Shell::SubstituteEnvironmentVariables(std::string_view command)
 		}
 		closing_percent += 1;
 
-		const std::string env_key(command.substr(1, closing_percent - 1));
-		if (const auto env_val = GetEnvStr(env_key)) {
-			expanded += env_val->substr(env_key.length() + sizeof('='));
+		if (const auto env_val{
+		            GetEnvStr(command.substr(1, closing_percent - 1))}) {
```

Comment:
`=`

Please let's not micro-optimise and obfuscate the syntax for no good reason.